import-export-cli/cmd/addEnv.go

```diff
@@ -32,6 +32,7 @@ var flagDevPortalEndpoint string    // DevPortal endpoint of the environment to
 var flagRegistrationEndpoint string // registration endpoint of the environment to be added
 var flagApiManagerEndpoint string   // api manager endpoint of the environment to be added
 var flagAdminEndpoint string        // admin endpoint of the environment to be added
+var flagMiManagementEndpoint string // mi management endpoint of the environment to be added
```

Comment:
Shall we run "go run tools/gen.go" to generate the docs (.md files) to the whole feature and commit those files here?
Refer [1] "Generating docs" section.

[1] https://github.com/wso2/product-apim-tooling/blob/master/import-export-cli/README.md

Reply:
Fixed with e053d59

Reply:
Shall we add the commands and usage of this feature to the file in [2] as well?

[2] https://github.com/wso2/product-apim-tooling/blob/master/import-export-cli/resources/README.html

Reply:
Fixed with 49921b9